### PR TITLE
Replace deprecated Porter URIs

### PR DESCRIPTION
**Pull request by piotr-roslaniec**
**Type of PR:**
- Feature

**Required reviews:**
- 1

**Notes for reviewers:**
- Not sure about lynx: 
  - https://porter-lynx.nucypher.community/ works 
  - https://porter-lynx.nucypher.io/ doesn't work

### CODE REVIEW

File: packages/shared/src/porter.ts [hunk 1]
```diff
@@ -13,9 +13,8 @@ import { Base64EncodedBytes, ChecksumAddress, HexEncodedBytes } from './types';
 import { fromBase64, fromHexString, toBase64, toHexString } from './utils';
 
 const porterUri: Record<string, string> = {
-  mainnet: 'https://porter.nucypher.community',
-  tapir: 'https://porter-tapir.nucypher.community',
-  oryx: 'https://porter-oryx.nucypher.community',
+  mainnet: 'https://porter.nucypher.io',
+  tapir: 'https://porter-tapir.nucypher.io',
   lynx: 'https://porter-lynx.nucypher.community',
```
Comment (derekpierre) [inline packages/shared/src/porter.ts:18]: If we are switching, should we just switch all? i.e. also switch lynx - cc @KPrasch

Comment (piotr-roslaniec) [reply packages/shared/src/porter.ts:18]: Yes, but currently https://porter-lynx.nucypher.io/ doesn't work

Comment (derekpierre) [reply packages/shared/src/porter.ts:18]: Yep. I just mean that we shouldn't merge this before making that happen.

**Comment by derekpierre** [comment]
Will be fixed via #555 .

**Pull request closed without merging**